errata.tex

```diff
@@ -732,6 +732,10 @@
   & 897-g94fb722
   & In~\ref{item:sigcmp}, ``if $f:\hom_X(x,y)$'' should be ``if $f:\hom_X(x,y)$ and $g:\hom_X(y,z)$''.\\
   %
+  \cref{sec:sip}
+  & merge of 988c807ca5b23a019536ff446763ba035062930c
```

Comment:
This should be a comment: `& % merge of ...`.

Reply:
fixed